README.md

```diff
@@ -5,9 +5,8 @@
 
 **HACKTOBERFEST TIME**
 
-## Enter your name and a favourite quote below.
+## SURAJ BRANWAL
```

Comment:
Please see the README.md again, @brandier-jd . Get inspiration from how others did it.